docs/components/servo/_index.md

```diff
@@ -6,18 +6,29 @@ type: "docs"
 description: "A hobby servo is a special type of small motor whose position you can precisely control."
 tags: ["servo", "components"]
 icon: "/components/img/components/servo.svg"
+no_list: true
 # SME: #team-bucket
 ---
 
-The Viam *servo* component supports hobby servos: small motors with built-in closed-loop control.
-Servos can be useful in robotics because their position is easily and precisely controlled.
+{{% alert title="Tip" color="tip" %}}
 
-- The servo component does not support servomotors.
-To configure an industrial servomotor, use the [motor](/components/motor/) component with an [encoder](/components/encoder/).
+The Viam servo component supports [hobby servos](https://learn.adafruit.com/adafruit-motor-selection-guide/rc-servos).
 
-Example wiring diagram for a servo wired to a Raspberry Pi board:
+If you have another type of servomotor, it is probably coupled with an [encoder](/components/encoder), not potentiometer, for position feedback.
```

Comment:
so are hobby motors always with a potentiometer? Would it be correct to say The Viam servo component supports hobby servos that have potentiometers?

Reply:
@npentrel I *think* that is the case, but should confirm with @randhid-- Rand, is this distinction accurate? and if so do you think the sensor specifics for "Viam servos" is important info to include?

Reply:
hobby *servos* that we support in both drivers always have a potentiometer, have a potentiometer. Motors can have many encoder types. There are non-pin based servos that exist with encoders that we do not support yet. This sentence is very specific to `gpio` and `pi` which are the only ones that exist in rdk.